.github/workflows/release.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  repository_dispatch:
+
+jobs:
+  release:
+    if: ${{ github.event.client_payload.passed }}
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ github.event.client_payload.version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set version
+        run: |
+          sed -i "s/version = \".*\"/version = \"$VERSION\"/g" recipe/meta.yaml
```

Comment:
You'll also need to ensure that the `build_number` is reset to `0` when auto-bumping the version